libs/core/src/lib/rating-indicator/components/rating-indicator.component.ts

```diff
@@ -0,0 +1,373 @@
+
+interface NumberKey<T> {
+    [key: number]: T;
+}
+
+interface RatingViewItem {
+    rate: number;
+    votes: number;
+}
+
+@Component({
+    selector: 'fd-rating-indicator',
+    templateUrl: './rating-indicator.component.html',
+    styleUrls: [
+        './rating-indicator.component.scss',
+        /*
+        * NOTE: Will be remove this style after merge PR with fix for this issue:
+        * Issue link: https://github.com/SAP/fundamental-styles/issues/1753
+        * PR link: https://github.com/SAP/fundamental-styles/pull/1782
+        */
+        './rating-indicator.css'
+    ],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR,
+            useExisting: forwardRef(() => RatingIndicatorComponent),
+            multi: true
+        }
+    ]
+})
+export class RatingIndicatorComponent implements OnInit, OnChanges, CssClassBuilder, ControlValueAccessor {
+    /** User's custom classes */
+    @Input()
+    class: string;
+
+    /** Sets [name] property of input. */
+    @Input()
+    name: string;
+
+    /** User's aria-label attr */
+    @Input()
+    @HostBinding('attr.aria-label')
+    ariaLabel: string;
+
+    /**
+     * Whether the rating indicator is disabled
+     */
+    @Input()
+    @HostBinding('class.is-disabled')
+    @HostBinding('attr.aria-disabled')
+    disabled = false;
+
+    /**
+     *
+     * Whether the rating indicator is in displayMode
+     */
+    @Input()
+    @HostBinding('class.is-display-mode')
+    displayMode = false;
+
+    /**
+     * Number of stars/icons/images to display. Max value is 7 and min value is 3
+     */
+    @Input()
+    set indicatorCapacity(value: number) {
+        const val = coerceNumberProperty(value, INDICATOR_DEFAULT_CAPACITY);
+        this._indicatorCapacity = Math.max(INDICATOR_CAPACITY_RANGE.min, Math.min(INDICATOR_CAPACITY_RANGE.max, val));
+    }
+
+    /**
+     * Whether or not to display half values.
+     */
+    @Input()
+    allowHalves = false;
+
+    /**
+     * User's value number of ratings. If provided, Overrides ratingAverage.
+     */
+    @Input()
+    value = 0;
+
+    /**
+     * Total number of ratings. If provided, will display text showing the total number of ratings.
+     */
+    @Input()
+    totalRatings: number;
+
+    /**
+     * Rating average
+     */
+    @Input()
+    ratingAverage: number;
+
+    /**
+     * Object containing key-value pairs where the key is the rating and the value is the total sum of those ratings.
+     * Overrides totalRatings and ratingAverage.
+     */
+    @Input()
+    ratings: NumberKey<number>;
+
+    /**
+     * Whether or not to display the popover that shows the sum of each rating. Requires [ratings] object.
+     */
+    @Input()
+    displaySumPopover = false;
+
+    /**
+     * Icon class from fundamental-styles lib https://sap.github.io/fundamental-styles/?path=/docs/components-icon--sizes
```

Comment:
I think we should simplify to: "Rating icon class". Developers can use custom icons outside of Fundamental Styles library.

Reply:
Yep, but tech doc and style for this component is not supported a simple way for a custom icon. We need to change the structure and in my opinion, it would be simpler with two templates for a rated and unrated icon (for example 
![image](https://user-images.githubusercontent.com/38053194/98390721-6ea12000-205e-11eb-8db5-2dd9e03d81de.png)
)